src/main/java/com/exasol/adapter/document/edml/ToDecimalMapping.java

```diff
@@ -0,0 +1,27 @@
+package com.exasol.adapter.document.edml;
+
+import com.exasol.adapter.document.mapping.MappingErrorBehaviour;
+
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+import lombok.extern.jackson.Jacksonized;
+
+/**
+ * Java representation of the EDML toDecimalMapping.
```

Comment:
```suggestion
 * Java representation of the EDML that map to {@code DECIMAL} column.
```

Reply:
`toDecimalMapping` is the keyword in the EDML. So I'd like to mention it here. How about using `{@code toDecimalMapping}`?

Reply:
Yes, that is fine.